### PR TITLE
Feature/implement user info container

### PR DESCRIPTION
**Pull request by ArtemLigerko**
WM-Screenshots-20220818155722

### CODE REVIEW

File: src/containers/UserInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+      <Card.Body>
+        <Row xs="auto" className="justify-content-end m-0">
+          <Button
+            variant="light"
+            className="p-2 justify-content-center align-items-center rounded-pill border border-1"
+            type="button"
+          >
+            Змінити профіль
+          </Button>
+        </Row>
```
Comment (KirillSerg) [inline src/containers/UserInfo.tsx:63]: bg color of button is white (or it has opacity) in original

**Review by KirillSerg — APPROVED** [review]
cool

**Comment by alexandrtovmach** [comment]
conflicts

**Comment by ArtemLigerko** [comment]
> conflicts

resolved

File: src/containers/UserInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  flagLogo: string;
+  userNickname: string;
+  aboutMeText: string;
+  userLocation: string;
+  userEmail: string;
+  userBirthday: string;
+  registrationDate: string;
+  inReadersCount: number;
+  readersCount: number;
+}
```
Comment (VitaliiDenysiuk) [inline src/containers/UserInfo.tsx:25]: I think some of these fields will not always be present. 
I have a suggestion to mark some of them "?". 
it means that this field is optional. Sometimes we do not show one of this field. For example email can be hidden from other users, but can be showen for you. 
Or userLogo may be missing

File: src/containers/UserInfo.tsx [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+  inReadersCount,
+  readersCount,
+}) => {
+  return (
+    <Card className="rounded-0 border-0">
+      <Card.Img variant="top" className="rounded-0" src={headLogo} />
+
+      <StyledCardBody className="p-0">
+        <img
+          className="rounded-circle ms-3"
```
Comment (Mykyta-Pysarev) [inline src/containers/UserInfo.tsx:47]: move width and height to styled component

**Review by Mykyta-Pysarev — APPROVED** [review]

File: src/containers/UserInfo.tsx (hunk 3 above)
Comment (ArtemLigerko) [reply src/containers/UserInfo.tsx:47]: is it necessary to create styled const only for width and height?

**Review by alexandrtovmach — APPROVED** [review]

**Pull request merged**